e2e/framework/provisioning/runner_linux.go

```diff
@@ -31,3 +63,22 @@ func (runner *LinuxRunner) Copy(local, remote string) error {
 }
 
 func (runner *LinuxRunner) Close() {}
+
+func (runner *LinuxRunner) Logf(format string, args ...interface{}) {
```

Comment:
Should we give `LinuxRunner` a [`testlog`](https://github.com/hashicorp/nomad/blob/master/helper/testlog/testlog.go)?

Having the `NOMAD_TEST_STDOUT` env var option might be nice since thats usually what we use to dump logs to stdout.

Reply:
The reason for the way I did the ssh runner was so that we'd get logs while the provisioning process was running, and not just when it was complete as one does for normal test logging in go (even with the `-v` flag, the logs only get dumped at the end of the run). But I'm definitely open to improvements on that!